app/konnect/runtime-manager/runtime-instances/renew-certificates.md

```diff
@@ -44,20 +44,15 @@ generate new certificates and replace them on the existing nodes.
 
 ### Generate new certificates
 
-1. Open {% konnect_icon runtimes %} **Runtime Manager**, select a runtime group, and
-click **Create Runtime Instance**.
-2. Click the tile that suits your environment: **Linux** or **Kubernetes**.
+You can generate new certificates from the {% konnect_icon runtimes %} **Runtime Manager**, using the **certificates** configuration page. 
```

Comment:
There are two certificates pages with Runtime Group - one is the certificates tab on the side navigation and the other one is dataplane certificates option inside the runtime group actions. 

This feature is for the dataplane certificates inside the runtime group actions. Can we call out the navigation specifically instead of referring to just the certificates configuration page as there is scope of confusion in that terminology?